tests/get-identity-apikey-only-rpc.test.ts

```
@@ -0,0 +1,48 @@
+  })
+
+  it.concurrent('allows service role RPC access', async () => {
+    const supabaseServiceRole = createClient<Database>(SUPABASE_URL, SUPABASE_SERVICE_KEY, {
+      global: { headers: { capgkey: APIKEY_TEST_ALL } },
+    })
+
+    const { data, error } = await supabaseServiceRole.rpc('get_identity_apikey_only', { keymode: keyModes })
+
+    expect(error).toBeNull()
+    expect(typeof data).toBe('string')
+    expect((data as string)).toMatch(/^[0-9a-fA-F-]{36}$/)
```

ESLint

[error] The character class(es) '[0-9a-fA-F-]' can be simplified using the `i` flag.

(regexp/use-ignore-case)